### ISSUE TITLE
[ECP-8713] Building assets of Administration and Storefront without a Database

### ISSUE THREAD

**Issue by wannevancamp** [Adyen/adyen-shopware6#430]
**Describe the bug**
It is common to prebuild assets in professional deployments to deploy the build artefact assets to the production environment. This task is mostly done by a CI job that doesn't have access to the production database. 
Shopware has foreseen a config for this: https://developer.shopware.com/docs/guides/hosting/installation-updates/deployments/build-w-o-db.html

Plugins can be compiled in the CI and be synced to the production server without a database or even a working Symfony Container. 

This plugin builds assets when the container builds and therefore can't be used in CI builds without a database or Symfony Container.

```
public function build(ContainerBuilder $container): void
    {
        parent::build($container);
        $this->installJsAssets($container->getParameter('kernel.shopware_version'));
    }
```

Because the assets doesn't exist in the production environment the following error is thrown:
```
Warning: copy(/.../.../.../vendor/store.shopware.com/adyenpaymentshopware6/src/Resources/public/administration/js/adyen-payment-shopware64.js.dist): Failed to open stream: No such file or directory
```

**Versions**
Shopware version: all versions
Plugin version: all versions

**To Reproduce**
1. Install plugin
2. Setup build CI without a database: https://developer.shopware.com/docs/guides/hosting/installation-updates/deployments/build-w-o-db.html
3. Open frontend

**Expected behavior**
Assets should be compiled beforehand or when using the bin/build-storefront.js and bin/build-administration.js

**Comment by peterojo** [comment]
Hi @wannevancamp!
Thanks for bringing this to our attention! We'll investigate this option and decide on the best way to make it easier for the majority of the plugin users.

**Comment by sneakyvv** [comment]
This means the plugin is simply NOT working on SW 6.5

Can we bump the priority on this?

**Comment by wannevancamp** [comment]
@MelvinAchterhuis , I've just verified. The issue occurs with both versions.
Did you run `bin/build-administration.sh` and cleared the cache (`rm -rf var/cache/`) afterwards?

**Comment by wannevancamp** [comment]
@peterojo, meanwhile I found why some merchants experience this issue and others do not.
This issue only occurs between the following Shopware versions 6.5.1.0 - 6.5.4.0.
Besides that, this can be fixed for those versions. The compatibility should be updated so that merchants know that the extension is not compatible with some versions: https://store.shopware.com/en/adyen70424242359f/adyen-payments-for-shopware-6.html

If there are no plans to fix this issue for those versions, this issue can be closed.

**Comment by peterojo** [comment]
Hi all! 
Adding a fix in #439 to fail safely if the copy process is not possible.
Note that the plugin already ships with the generated files for Shopware 6.5, so that running the build-*.sh command should not be necessary for the plugin assets to work.
